ex1/functions.py

```diff
@@ -65,23 +116,125 @@ def f():
 # compare against. The function should return a list with equal length as the
 # input list containing for each number in the original list either True or
 # False if the number was greater than or equal to n.
+def check_list(list_of_numbers, n):
+	"""
+	Parameters
+	----------
+	list_of_numbers : list of int
+	n : int
+
+	Returns
+	-------
+	list of bool
+			True for every element in list_of_numbers greater than n, else False
+	
+    	"""
+	bool_list = [] # list with boolen values
+	for i in list_of_numbers:
+		if(i>=n):
```

Comment:
You do not need an `if` `else` clause here since you can fill the list directly with `i>=n`. See also #20